[DOC] S3 API 가이드 보강

minkyu-lee-nhn opened the pull request:
- #tc-iaas-obs/213
- https://nhnent.dooray.com/project/posts/3119667992908048797

minkyu-lee-nhn committed:
Add contents of AWS S3 SDK (.NET, Python, Java)

minkyu-lee-nhn committed:
검수 의견 반영
- 전수 검사
- 예외 처리
- 컨텍스트 일반화

minkyu-lee-nhn committed:
[S3 API 가이드] [AWS CLI - 오브젝트 업로드] 설명 보충 (Draft)

minkyu-lee-nhn committed:
[S3 API 가이드] [AWS CLI - 오브젝트 업로드] 설명 보충 (Draft)

minkyu-lee-nhn committed:
[S3 API 가이드] [AWS CLI - 오브젝트 업로드] 설명 보충 (Draft)

minkyu-lee-nhn committed:
[S3 API 가이드] [AWS CLI - 오브젝트 업로드] 설명 보충 (Draft)

minkyu-lee-nhn committed:
[S3 API 가이드] [AWS CLI - 오브젝트 업로드] 설명 보충 (Draft)

minkyu-lee-nhn committed:
S3 API(버킷, 오브젝트)의 응답 내용 수정
- SDK로부터 가공된 데이터 제거
- S3 공통 헤더 생략
- 본문 반환 여부 및 기준 상태 코드 코멘트 추가
- 본문 예시 추가 (XML)

minkyu-lee-nhn committed:
s3-api-guide 수정
- AWS SDK 언어별 내용 추가
- AWS CLI 오브젝트 업로드 내용 일부 수정

jhjang04 commented on ko/s3-api-guide.md:
````diff
@@ -937,8 +878,266 @@ public class AwsSdkExapmple {
+
+<details>
+<summary>.NET SDK 클라이언트 클래스</summary>
+```csharp
````
summary 뒤에 한줄 띄워줘야 md 파일 파싱이 똑바로 되더라구요.
요거 가이드에 적용될떈 어떨지 모르겠습니다.

minkyu-lee-nhn replied:
@jhjang04 

말씀 대로 파서에 따라 summary 태그와 code block 사이에 개행이 있어야 적용이 되는 경우가 있고 6520f14 기준 개행 추가되었습니다.

minkyu-lee-nhn committed:
S3 API 가이드 개정
- "container"를 "bucket"으로 변경
- multipart upload core specification 전문 삭제
- MPU default threshold 명시 및 part size를 추상적으로 표현
- 숫자와 단위 사이 공백 삭제
- SDK 설치 관련 내용 삭제
- 예제의 버킷, 오브젝트 이름을 단순하게 변경
- role, user 이름을 유저스럽게 변경
- 결과에 영향을 주지 않는 garbage와 오타 정리 (^\ +$, AwsSdkExapmple)
- 번역체 순화
- 문장 단순화

minkyu-lee-nhn committed:
오타 정정
- 공백 문자가 두 개 들어가있는 오타 정정

minkyu-lee-nhn committed:
unordered list에 plus 대신 asterisk 사

minkyu-lee-nhn committed:
sample code snippet에 line wrapping 적용 (cc 80)

minkyu-lee-nhn committed:
read the docs에서 렌더링되지 않는 markdown syntax element를 html tag로 변환
- unordered list (margin, padding 0)
- code (display inline)

minkyu-lee-nhn committed:
C# 샘플 코드 스타일을 좀 더 일관되게 수정

minkyu-lee-nhn committed:
AWS CLI - 멀티파트 업로드 설명 수정
- 일반적으로 사용자가 지정하지 않거나 할 수 없는 파라미터(멀티파트 업로드 임계치, 파트 사이즈)에 대한 설명을 생략하고 추상적으로 표현
- AWS에서 통용되는 용어 준수 (세그먼트->파트, 매니페스트->오브젝트)
- 예상 가능한 내용에 대한 부연 설명 생략 (오브젝트 경로)

minkyu-lee-nhn committed:
C# 샘플 코드 스타일을 좀 더 일관되게 수정

minkyu-lee-nhn committed:
리뷰 의견 반영
- MPU 설명부를 ul 대신 quote로 대체 (RTD 파싱 이슈로 html tag 사용)
- code language가 지정되지 않은 code block에 loanguage 지정

minkyu-lee-nhn committed:
gov, ngsc에 수정분 반영

minkyu-lee-nhn committed:
변경된 ngsc 도메인 주소 반영

minkyu-lee-nhn committed:
Merge branch 'develop' into tc-iaas-obs/213

minkyu-lee-nhn committed:
S3에서 '컨테이너'로 표현한 부분을 '버킷'으로 정정

minkyu-lee-nhn committed:
Revert "S3에서 '컨테이너'로 표현한 부분을 '버킷'으로 정정"

This reverts commit 72d3a5e260e16ef3b072fadda8cec332e24ea622.

minkyu-lee-nhn committed:
s3api에 한해서 '컨테이너'를 '버킷'으로 표현
버킷 조회, 버킷 목록 조회의 응답 단락의 설명을 구체화
